### DISCUSSION TITLE
License support

### DISCUSSION THREAD

**Discussion by vthiru86** [nathanwoulfe/Plumber-2#75] [Q&A]
Hi,

My organisation purchased a license for Plumber at https://thisredwolf.com/ earlier this month but have not received the tax invoice via email yet.

Is there somewhere where we can source the tax invoice or are there any details for a support contact we can follow this up with?

**Comment by nathanwoulfe** [comment]
Hi 👋

The invoice should have been sent automatically via Stripe, but I've had issues with that in the past. I'll sort it out for you.

**Reply by vthiru86** [reply]
Thanks @nathanwoulfe, is there a way to PM you the details related to the transaction?

**Reply by nathanwoulfe** [reply]
Absolutely - nathan@nathanw.com.au
